pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>virtual-schema-common-java</artifactId>
-    <version>2.1.0</version>
+    <version>2.2.0</version>
```

Comment:
2.2.1

Reply:
It's an added feature -> increase minor, reset fix.